### PR TITLE
Update for SmartPy 0.20.0

### PR DESCRIPTION
**Pull request by timothymcmackin**
Related to https://github.com/trilitech/tutorial-applications/pull/14.

Related tutorial updated in https://github.com/trilitech/tezos-developer-docs/pull/487.

### CODE REVIEW

File: simple-nft-svelte/contract/simple-nft.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 @sp.module
 def main():
+    import sp.utils as utils
```
Comment (wyn) [inline simple-nft-svelte/contract/simple-nft.py:9]: this isn't needed

File: simple-nft-svelte/contract/simple-nft.py [hunk 2]
```diff
@@ -228,9 +230,9 @@ def make_metadata(symbol, name, decimals):
         """Helper function to build metadata JSON bytes values."""
         return sp.map(
             l={
-                "decimals": sp.utils.bytes_of_string("%d" % decimals),
-                "name": sp.utils.bytes_of_string(name),
-                "symbol": sp.utils.bytes_of_string(symbol),
+                "decimals": utils.bytes_of_string("%d" % decimals),
```
Comment (wyn) [inline simple-nft-svelte/contract/simple-nft.py:233]: so now we are in normal python code - not SmartPy code - and you may hear us talking about 'being in a test_scenario' or 'scenario code' or similar, we mean normal python code.  Anyway, the `bytes_of_string` function and `metadata_or_url` function are in `scenario_utils` now - I think this file is a bit old? - so we need `sp.scenario_utils.bytes_of_string`.

But note that is just pure python, not SmartPy.

File: simple-nft-svelte/contract/simple-nft.py [hunk 3]
```diff
@@ -242,16 +244,16 @@ def make_metadata(symbol, name, decimals):
 
     @sp.add_test(name="Minimal test")
     def test():
-        scenario = sp.test_scenario(main)
+        scenario = sp.test_scenario("test_simple_nft", main)
         c1 = main.Fa2NftMinimal(
-            sp.utils.metadata_of_url("https://cloudflare-ipfs.com/ipfs/QmW8jPMdBmFvsSEoLWPPhaozN6jGQFxxkwuMLtVFqEy6Fb")
+            utils.metadata_of_url("https://cloudflare-ipfs.com/ipfs/QmW8jPMdBmFvsSEoLWPPhaozN6jGQFxxkwuMLtVFqEy6Fb")
```
Comment (wyn) [inline simple-nft-svelte/contract/simple-nft.py:249]: see comment above, needs to be `sp.scenario_utils.metadata_of_url`

File: simple-nft-svelte/contract/simple-nft.py [hunk 4]
```diff
@@ -242,16 +244,16 @@ def make_metadata(symbol, name, decimals):
...
         )
         scenario += c1
 
     from templates import fa2_lib_testing as testing
 
     c1 = main.Fa2NftMinimalTest(
-        metadata=sp.utils.metadata_of_url("https://cloudflare-ipfs.com/ipfs/QmW8jPMdBmFvsSEoLWPPhaozN6jGQFxxkwuMLtVFqEy6Fb"),
+        metadata=utils.metadata_of_url("https://cloudflare-ipfs.com/ipfs/QmW8jPMdBmFvsSEoLWPPhaozN6jGQFxxkwuMLtVFqEy6Fb"),
```
Comment (wyn) [inline simple-nft-svelte/contract/simple-nft.py:256]: see comment above, needs to be `sp.scenario_utils.metadata_of_url`

File: simple-nft-svelte/contract/simple-nft.py (hunk 2 above)
Comment (timothymcmackin) [reply simple-nft-svelte/contract/simple-nft.py:233]: >  think this file is a bit old?
I guess I should rewrite it or use an updated file that uses FA2lib.

**Comment by timothymcmackin** [comment]
@wyn Thanks for the initial review. I redid the contract using FA2lib with one hack: I want anyone to be able to mint an NFT, so I mocked the is_administrator function. How does this look now?

**Comment by wyn** [comment]
The contract looks good - apart from that one little thing about inheritence ordering - I also ran the contract with a fresh smartpy-0.20.0 install and it worked.

**Comment by timothymcmackin** [comment]
> I'm not massively knowledgeable about svelte but did you try running it locally at all?

Yes, it works for me locally.

**Pull request merged**